buildout.d/solr.cfg

```diff
@@ -38,6 +38,7 @@ index =
     name:Description            type:text copyfield:default copyfield:spelling stored:true
     name:SearchableText         type:text copyfield:default copyfield:spelling stored:false
     name:tags                   type:string copyfield:default stored:true multivalued:true
+    name:Subject                type:string stored:true indexed:false multivalued:true
```

Comment:
We shouldn't need both tags and Subject. The ploneintranet.search code pulls the tag values out of plone's 'Subject' field already.